src/main/java/calculator/model/delimiter/CustomDelimiter.java

```diff
@@ -0,0 +1,46 @@
+package calculator.model.delimiter;
+
+import static calculator.model.delimiter.DefaultDelimiter.DEFAULT_DELIMITERS;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+public class CustomDelimiter implements Delimiter {
+    public static final String CUSTOM_DELIMITER_SUFFIX = "\\n";
+    private static final String CUSTOM_DELIMITER_PREFIX = "//";
+    private final Set<String> delimiters;
+
+    protected CustomDelimiter(final String input) {
+        this.delimiters = initDelimiters(input);
+    }
+
+    private Set<String> initDelimiters(final String input) {
+        Set<String> delimiters = new HashSet<>(Arrays.asList(DEFAULT_DELIMITERS));
+        delimiters.add(findCustomDelimiter(input));
+        return delimiters;
+    }
+
+    private String findCustomDelimiter(final String input) {
+        final int delimiterStart = input.indexOf(CUSTOM_DELIMITER_PREFIX);
+        final int delimiterEnd = input.indexOf(CUSTOM_DELIMITER_SUFFIX);
+        validateCustomDelimiterPosition(delimiterStart, delimiterEnd);
+        return escapeMetaCharacter(input.substring(delimiterStart + 2, delimiterEnd));
+    }
+
+    private void validateCustomDelimiterPosition(final int delimiterStart, final int delimiterEnd) {
+        if (delimiterEnd == -1 || delimiterStart == -1) {
+            throw new IllegalArgumentException("커스텀 구분자는 //와 \\n 사이에 위치하여야 합니다.");
```

Comment:
`//`와 `\\n` 대신 선언해 놓은 상수를 넣으면 좋을 것 같습니다!

Reply:
비지니스 로직에 큰 영향이 없는 단순한 오류 메세지라 따로 상수를 사용하지 않았습니다. 오류 메세지에도 상수를 활용하는 것이 좋을까요?🧐

Reply:
@kgy1008 
구분자를 `,`와 `:` 이 아닌 다른 문자로 변경해야 할 상황이 생기면 예외 문자 안에서도 저 기호들도 따로 변경해줘야 하는 불편함이 따를 것 같아서요!
```suggestion
            throw new IllegalArgumentException("커스텀 구분자는" + CUSTOM_DELIMITER_PREFIX + "와 " + CUSTOM_DELIMITER_SUFFIX +  "사이에 위치하여야 합니다.");
```
이렇게 바꾸면 일일이 변경을 줄일 수 있을 것 같습니다

Reply:
정말 좋은 의견 감사합니다!! 변경 가능성 측면을 생각해보니 상수를 이용하는 것이 좋네요:)